### PR TITLE
LPD-15231 Escape account name order details

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-15231

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:test:playwright

File: modules/test/playwright/pages/commerce/commerceAdminOrdersPage.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		value: number | string,
+		strictEqual?: boolean
+	) => Promise<{column: Locator; row: Locator}>;
+	readonly ordersTableRowLink: (orderId: number | string) => Promise<Locator>;
+	readonly page: Page;
+
+	constructor(page: Page) {
+		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+		this.ordersTable = page.locator(
+			'#_com_liferay_commerce_order_web_internal_portlet_CommerceOrderPortlet_fm .dnd-table'
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/pages/commerce/commerceAdminOrdersPage.ts:53]: why don't you use data-qa-id here?

File: modules/test/playwright/tests/commerce/commerceAdminOrderDetails.spec.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	);
+
+	const cart = await apiHelpers.headlessCommerceDeliveryCart.postCart(
+		{
+			accountId: account.id,
+			cartItems: [
+				{
+					options: '[]',
+					quantity: 1,
+					replacedSkuId: 0,
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerceAdminOrderDetails.spec.ts:51]: is this necessary?

File: modules/test/playwright/tests/commerce/commerceAdminOrderDetails.spec.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+}) => {
+	const site = await apiHelpers.headlessAdminUser.getSiteByFriendlyUrlPath(
+		'guest'
+	);
+
+	const channel = await apiHelpers.headlessCommerceAdminChannel.postChannel({
+		siteGroupId: site.id,
+	});
+
+	const catalog = await apiHelpers.headlessCommerceAdminCatalog.postCatalog();
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerceAdminOrderDetails.spec.ts:29]: why do you need to create a new catalog, can't you use the Master Catalog?

**Review by andrea-ale-sbarra — COMMENTED** [review]
few questions to answer before proceeding

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:stop

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**